lib/sequenceserver/blast/error.rb

```diff
@@ -0,0 +1,76 @@
+require 'English'
+
+require_relative 'constants'
+require_relative 'exceptions'
+
+module SequenceServer
+
+  #BLAST module
+  module BLAST
+    class Error
+      def initialize(job)
+        @job = job
+        callError
+      end
+
+      attr_reader :job
+
+      def callError
+        case job.exitStatus
+        when 1 # error in query sequence or options; see [1]
+          validate_options job.advanced_params # if advanced_params are incorrect
```

Comment:
I don't understand why advanced params is being validated here and why is it done twice (once when a job is created and here - after job is done).